docs/advanced.md

```diff
@@ -221,6 +221,28 @@ with httpx.Client(headers=headers) as client:
     ...
 ```
 
+## Monitoring download progress
+
+If you need to monitor download progress of large responses, you can use stream and the `response.last_raw_chunk_size` property.
+
+For example, showing a progress bar using the [`tqdm`](https://github.com/tqdm/tqdm) library while a response is being downloaded could be done like this…
+
+```python
+import tempfile
+
+import httpx
+from tqdm import tqdm
+
+with tempfile.NamedTemporaryFile() as download_file:
+    data = b"@" * 1000000
+    with httpx.stream("POST", "https://httpbin.org/anything", data=data) as response:
+        content_length = int(response.headers["Content-Length"])
+        with tqdm(total=content_length) as progress:
```

Comment:
I'm interested what arguments we need to display here to have tqdm display this using B, KB, MB, GB etc...

From https://github.com/tqdm/tqdm#extra-cli-options is sounds like the API might be this?...

```python
with tqdm(total=total, unit_scale=True, unit_divisor=1024, unit='B') as progress:
    ...
```

But it'd need testing to be sure.

Reply:
I checked this example, it looks fine, something like that:
```
100%|██████████████| 100M/100M [01:54<00:00, 918kB/s]
```